components/application-operator/cmd/manager/manager.go

```diff
@@ -55,14 +61,21 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = application_mapping_scheme.AddToScheme(mgr.GetScheme())
+	err = service_instance_scheme.AddToScheme(mgr.GetScheme())
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Printf("Preparing Helm Client.")
+
+	helmClient, err := kymahelm.NewClient(options.tillerUrl, options.helmTLSKeyFile, options.helmTLSCertificateFile, options.tillerTLSSkipVerify, options.installationTimeout)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	log.Printf("Preparing Release Manager.")
 
-	releaseManager, err := newReleaseManager(options, cfg)
+	releaseManager, err := newReleaseManager(options, cfg, helmClient)
```

Comment:
The name `ReleaseManager` is now confusing because Gateway is also deployed with Helm so it also has releases. I would suggest renaming it to `ApplicationReleaseManager`.